weather-app/src/Header.js

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import GetResponse from './ResponseApi';
+import GetGeo from './GeolocationApi';
+
+const Header = (props)=>{
+    const[city, setCity]= useState('')
+    const [checkCityValue, setCheck]=useState(true)
+    const {response, setResponse, SetHistory, history} = props
+
+    const historyHandler = () => {
+        SetHistory(!history)
+    }
+    async function clickHandler(){
+        if(city){
+            setResponse(await GetResponse(city))
+            setCity('')
+        }else{
+            setCheck(false)
+        }
+    }
+    const changeHandler =(event)=>{
+        setCity(event.target.value)
+        setCheck(true)
+    }
+     async function geolocationHandler(){
+     await GetGeo(setResponse);
+    }
```

Comment:
GetGeo не возвращает промис
```suggestion
     function geolocationHandler(){
        GetGeo(setResponse);
    }
```